core-kotlin-modules/core-kotlin-collections-6/src/test/kotlin/com/baeldung/parallelOperationsCollections/ParallelOperationCollectionsUnitTest.kt

```diff
@@ -105,9 +99,12 @@ class ParallelOperationCollectionsUnitTest {
         val observable = Observable.fromIterable(people)
             .flatMap(
                 {
-                    Observable.just(it).subscribeOn(Schedulers.computation()).doOnNext { person ->
-                        person.setAdult()
-                    }
+                    Observable.just(it)
+                        .subscribeOn(Schedulers.computation())
+                        .doOnNext { person ->
+                            person.setAdult()
+                            Thread.sleep(1500)
+                        }
```

Comment:
Besides the sleep call, is this just a reformatting?

Reply:
Yes, that's right. This is to make it easier to read.